pandas/tests/io/test_pytables.py

```diff
@@ -4999,6 +4999,62 @@ def test_read_nokey_empty(self):
             store.close()
             pytest.raises(ValueError, read_hdf, path)
 
+    # GH10143
+    def test_walk(self):
+        objs = {
+            'df1': pd.DataFrame([1, 2, 3]),
+            'df2': pd.DataFrame([4, 5, 6]),
+            'df3': pd.DataFrame([6, 7, 8]),
+            'df4': pd.DataFrame([9, 10, 11]),
+            's1': pd.Series([10, 9, 8]),
+            'a1': np.array([[1, 2, 3], [4, 5, 6]])
+        }
+
+        with ensure_clean_store('walk_groups.hdf', mode='w') as store:
+            store.put('/first_group/df1', objs['df1'])
```

Comment:
add a table or 2

Reply:
By table do you mean store one of the dataframes in table format?

Reply:
added